src/services/CohortModuleService.ts

```diff
@@ -0,0 +1,37 @@
+import { UpdateCohortModuleRequest } from 'src/types/Module';
+
+import HttpService, { Response } from './HttpService';
+
+const COHORT_MODULE_ENDOINT = '/api/v1/accelerator/cohorts/{cohortId}/modules/{moduleId}';
+
+/**
+ * Delete cohort module
+ */
+const deleteOne = async (moduleId: number, cohortId: number): Promise<Response> =>
```

Comment:
nit: can just be delete

Reply:
I get an error: 'delete' is not allowed as a variable declaration name.

Reply:
Ahh that makes sense